scripts/run_tests.sh

```diff
@@ -82,21 +97,37 @@ create_docker_network() {
 }
 
 start_docker_db() {
-    docker run \
+    params=()
+
+    if [[ gpu_only -eq 1 ]] || [[ rbc_only -eq 1 ]];then
```

Comment:
@jclay this is why the rbc tests were failing, it wasnt appending the runtime flag for them. Your choice on the syntax here, might be slightly better to move this logic someplace where if we wanted to switch to cpu-only for RBC, it would only be a single area to change instead of hunting this line down.

Reply:
ahh good catch!